en/animation/marionette/animation-graph-basics.md

```diff
@@ -0,0 +1,221 @@
+
+We call the action that an object is in that plays a particular animation, such as standby, move, run, attack, etc., **state**. <br>
+In general, an object has multiple states and switches between them in a certain logical order to perform different actions; such switches are called [transitions](state-translation.md) in the animation graph. The conditions that need to be satisfied when a transition occurs are called **transition conditions**.
+
+The animation state machine, on the other hand, is used to manage and control each state and the transition between states on the object, similar to a flowchart, which we can edit visually directly in the [Animation Graph panel](animation-graph-panel.md). Currently an animation graph contains a state machine, when the state machine is in a state in the animation graph, the animation corresponding to that state will be played. Then, according to the pre-built animation flow chart, the state machine can automatically control the playback and switching of the skeleton animation, etc.
+
+## State Basics
+
+The states in the state machine are not only used to control the animation, but also include **Entry**, **Exit**, **Any**, **Sub-State Machine**, which are special states, because they do not carry animation, but are only used to mark the start and end of the state machine. To distinguish them from other states, we call them "pseudo-states" for now.
+
+The [sub-state machine](#sub-state-machine) is nested in the state machine and is used to degrade complex animation logic, see the **sub-state machine** section below for details. We generally refer to the topmost state machine of the animation graph as the **Top Level State Machine**, similar to the root node in the scene.
+
+The **Entry**, **Exit**, and **Any** are fixed units that exist in the state machine by default and cannot be deleted. They can be seen in the grid layout area when you open the animation graph to edit the state machine, and it should be noted that **Exit** only exists in **sub-state machines**.
+
+![animation-graph-panel](animation-graph-panel/animation-graph-panel.png)
+
+- **Entry**: marks the entry of the state machine. **Entry** can only be the source where the state transition occurs, not the target of the transition.
+
+- **Exit**: exits the state machine after the state transition to **Exit**. **Entry** can only be used as the target of a state transition, not as the source of the transition.
+
+    **Exit exists only in the sub-state machine**, because after exiting the sub-state machine, you need to return to the parent state machine to continue executing the next transition; if you exit the top-level state machine, the animation flow terminates and the character is stationary, whereas in general, the character is continuously moving. <br>If you need to make the character completely stationary, you can do so by turning off the animation controller component where the current animation graph is located.
```

Comment:
```suggestion
    **Exit exists only in the sub-state machine** because after exiting the sub-state machine, it is necessary to return to the parent state machine to continue executing the next transition. If the top-level state machine is exited, the animation flow terminates and the character is stationary, whereas in general, the character is continuously moving. <br>To make the character completely stationary, turn off the animation controller component where the current animation graph is located.
```